apps/meteor/client/views/room/contextualBar/RoomFiles/components/FileItem.js

```diff
@@ -10,7 +9,7 @@ import MenuItem from './MenuItem';
 const hoverClass = css`
 	&:hover {
 		cursor: pointer;
-		background-color: ${colors.n100};
+		background-color: ${Palette.surface['surface-tint'].toString()};
```

Comment:
```suggestion
		background-color: ${Palette.surface['surface-tint']};
```

Reply:
yes it will:
ts will complain if you are passing that as a `color` or `bg` Box prop.
in this case it's a css in js property value :)

Reply:
you're completely right!